Project/src/Utils/Utils.js

```diff
@@ -1,4 +1,5 @@
-import { isCommunicationUser, isPhoneNumber, isCallingApplication } from '@azure/communication-common';
+import { isCommunicationUserIdentifier, isPhoneNumberIdentifier, isCallingApplicationIdentifier } from '@azure/communication-common';
+import commo from '@azure/communication-common'
```

Comment:
What is commo and where are we using it?

Reply:
this was mistake and should be removed